apps/bandwagon/forms.py

```diff
@@ -143,6 +145,12 @@ def __init__(self, *args, **kw):
                 self.instance.type in amo.COLLECTION_SPECIAL_SLUGS):
             del self.fields['slug']
 
+    def clean(self):
+        if self.cleaned_data['your_name']:
+            raise forms.ValidationError(
+                'You\'ve been flagged as spam, sorry about that.')
```

Comment:
Because this string is never intended to be read by an end user and never shown in the UI, but only to trip up bots, I said that localisation is not needed.